packages/web/src/pages/settings-page/components/desktop/SettingsCard.module.css

```diff
@@ -14,6 +14,10 @@
   text-align: left;
 }
 
+.settingsCardFull {
+  flex: 1 1 calc(100% - var(--harmony-unit-3)); /* Leave room for gap */
```

Comment:
Whats the context here?

Reply:
just to make the appearance card full width like the design

Reply:
Wait but there wouldn't be a gap with the full width card right? I'm also confused by this